Sources/Mantle/Elaborate.swift

```diff
@@ -245,25 +251,6 @@ extension TypeChecker where PhaseState == ElaboratePhaseState {
     return t
   }
 
-  // Takes a Pi-type and replaces all it's elements with metavariables.
-  private func fillPiWithMetas(_ ty: Type<TT>) -> [Term<TT>] {
```

Comment:
Could you revert this change?  I thought I'd need it at the time but now I realize I don't.